app-engine/java/src/com/google/cloud/tools/intellij/appengine/java/cloud/BuildDeploymentSourceType.java

```diff
@@ -42,7 +43,10 @@
 public abstract class BuildDeploymentSourceType
     extends DeploymentSourceType<ModuleDeploymentSource> {
 
-  public BuildDeploymentSourceType(@NotNull String id) {
+  static final String PROJECT_ATTRIBUTE = "project";
```

Comment:
IDE project or cloud project? Could we clarify this a bit more? Same for version - which version is this?

Reply:
I renamed as follows to hopefully clear this up:

`PROECT_ATTRIBUTE` -> `CLOUD_PROECT_ATTRIBUTE`
`VERSION_ATTRIBUTE` - > `APP_ENGINE_VERSION_ATTRIBUTE`

Reply:
Awesome, thanks! Constant confusion with those projects. What do you think if we add an enhancement to refactor the method names for `AppEngineDeployable` for another PR to also clarify `setProjectName` and `setVersion`?

Reply:
I was thinking the same thing. will do. I'll keep it separte since it touches a lot of places